components/swatch/index.css

```diff
@@ -195,53 +238,66 @@ governing permissions and limitations under the License.
 
   overflow: hidden;
 
-  border-radius: var(--spectrum-swatch-border-radius);
+  border-radius: var(--mod-swatch-border-radius, var(--spectrum-swatch-border-radius));
 
-  background-color: var(--spectrum-colorcontrol-checkerboard-light-color);
+
+  /* Checkerboard fill */
+  /* Static checkerboard properties */
+  --spectrum-swatch-checkerboard-size: 8px;
+  --spectrum-swatch-checkerboard-background-offset: 0px;
+  --spectrum-swatch-checkerboard-dark-color: rgb(217, 217, 217);
```

Comment:
do we have a token for this? 
I could understand that the checker-board doesn't get yet a token but it's used across a lot of products. 
The place for wish-tokens is: https://paper.dropbox.com/doc/Missing-tokens-from-Core-Tokens-CSS-Migration--BnxEwsd_PVslzsEIgDEXHRiYAg-y9o9A3tt2qKFBSzUjfhsY